lib/bloc/input/input_bloc.dart

```diff
@@ -91,7 +89,8 @@ class InputBloc extends Cubit<InputState> {
     return InputState(protocol: command.protocol, inputData: invoice);
   }
 
-  Stream<DecodedClipboardData> get decodedClipboardStream => _device.clipboardStream.distinct().skip(1).map((clipboardData) {
+  Stream<DecodedClipboardData> get decodedClipboardStream =>
+      _device.clipboardStream.distinct().skip(1).map((clipboardData) {
```

Comment:
Is this a dart format change?

Reply:
yes this must have been automatic styling , I only removed the two cases. Do we use some specific style rules?